ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -268,13 +268,98 @@ - (void)getUserVideo:(NSDictionary *)constraints
+                     if ([[mandatoryParameters valueForKey:@"maxWidth"] isKindOfClass:[NSNumber class]]) {
+                         widthMax = [NSMutableString stringWithFormat:@"%@", (NSInteger)mandatoryParameters[@"maxWidth"]];
+                     }else{
+                         widthMax =mandatoryParameters[@"maxWidth"];
+                     }
+                     if(widthMax != nil){
+                         [mandatory setObject:widthMax forKey:@"maxWidth"];
+                     }
+                 }
+                 if([mandatoryParameters valueForKey:@"minFrameRate"] != nil) {
+                     // The key existed...
+                     NSMutableString *frameRateMin = nil;
+                     if ([[mandatoryParameters valueForKey:@"minFrameRate"] isKindOfClass:[NSNumber class]]) {
+                         frameRateMin = [NSMutableString stringWithFormat:@"%@", (NSInteger)mandatoryParameters[@"minFrameRate"]];
+                     }else{
+                         frameRateMin =mandatoryParameters[@"minFrameRate"];
+                     }
+                     if(frameRateMin != nil){
+                         [mandatory setObject:frameRateMin forKey:@"minFrameRate"];
+                     }
+                 }
+                 if([mandatoryParameters valueForKey:@"minHeight"] != nil) {
+                     // The key existed...
+                     NSMutableString *heightMin = nil;
+                     if ([[mandatoryParameters valueForKey:@"minHeight"] isKindOfClass:[NSNumber class]]) {
+                         heightMin = [NSMutableString stringWithFormat:@"%@", (NSInteger)mandatoryParameters[@"minHeight"]];
+                     }else{
+                         heightMin =mandatoryParameters[@"max"];
+                     }
+                     if(heightMin != nil){
+                         [mandatory setObject:heightMin forKey:@"minHeight"];
+                     }
+                 }
+
+                 if([mandatoryParameters valueForKey:@"minWidth"] != nil) {
+                     // The key existed...
+                     NSMutableString *widthMin = nil;
+                     if ([[mandatoryParameters valueForKey:@"minWidth"] isKindOfClass:[NSNumber class]]) {
+                         widthMin = [NSMutableString stringWithFormat:@"%@", (NSInteger)mandatoryParameters[@"minWidth"]];
+                     }else{
+                         widthMin =mandatoryParameters[@"minWidth"];
+                     }
+                     if(widthMin != nil){
+                         [mandatory setObject:widthMin forKey:@"minWidth"];
+                     }
+                 }
+                 betterConstraints = [[RTCMediaConstraints alloc] initWithMandatoryConstraints:mandatory optionalConstraints:nil];
+             }else{
+               betterConstraints = [self defaultMediaStreamConstraints];
+             }
+         }else{
+         betterConstraints = [self defaultMediaStreamConstraints];
```

Comment:
I dont think we should do this because the current constraints are just minimum, and not maximum, so you would have to nullify the opposite direction to make sure you weren't scoping your range too small, in some cases.  Ie you would have constraints that you didn't realize you had.